LaTeX/TeX.sublime-syntax

```diff
@@ -231,6 +229,55 @@ contexts:
     - include: brace-group-end
     - include: main
 
+
+###[ CHARACTER CODES ]##########################################################
+
+  character-codes:
+    - match: (\\)(?:cat|math|uc|lc|del|sf)code{{endcs}}
+      scope: keyword.control.character-code.tex
+      captures:
+        1 : punctuation.definition.backslash.tex
+      push:
+        - character-code-scope
+        - character-code-value
+        - character-code-assignment
+        - character-code-number
+
+  character-code-scope:
+    - meta_scope: meta.function.character-code.tex
+    - include: immediately-pop
```

Comment:
To follow suggestions of the previous PR ...

1. `character-code-meta`
2. `meta_include_prototype: false`

```suggestion
      push:
        - character-code-meta
        - character-code-value
        - character-code-assignment
        - character-code-number

  character-code-meta:
    - meta_include_prototype: false
    - meta_scope: meta.function.character-code.tex
    - include: immediately-pop
```